evalml/tests/utils_tests/test_gen_utils.py

```diff
@@ -391,3 +394,197 @@ def test_convert_to_woodwork_structure():
     X_expected = ww.DataTable(pd.DataFrame(X_np))
     pd.testing.assert_frame_equal(X_expected.to_dataframe(), _convert_to_woodwork_structure(X_np).to_dataframe())
     assert np.array_equal(X_np, np.array([[1, 2], [3, 4]]))
+
+
+@pytest.mark.parametrize("file_name,format,interactive",
+                         [
+                             ('test_plot', 'png', False),
+                             ('test_plot.png', 'png', False),
+                             ('test_plot.', 'png', False),
+                             ('test_plot.png', 'jpeg', False)
+                         ])
+def test_save_plotly_static_default_format(file_name, format, interactive, decision_tree_classification_pipeline_class, tmpdir, has_minimal_dependencies):
+    if not has_minimal_dependencies:
+        pipeline = decision_tree_classification_pipeline_class
+        feat_fig_ = pipeline.graph_feature_importance()
+
+        filepath = os.path.join(str(tmpdir), f'{file_name}') if file_name else None
+        no_output_ = save_plot(fig=feat_fig_, filepath=filepath, format=format, interactive=interactive, return_filepath=False)
+        output_ = save_plot(fig=feat_fig_, filepath=filepath, format=format, interactive=interactive, return_filepath=True)
+
+        assert not no_output_
+        assert os.path.exists(output_)
+        assert isinstance(output_, str)
+        assert os.path.basename(output_) == 'test_plot.png'
+
+
+@pytest.mark.parametrize("file_name,format,interactive",
+                         [
+                             ('test_plot', 'jpeg', False)
+                         ])
+def test_save_plotly_static_different_format(file_name, format, interactive, decision_tree_classification_pipeline_class, tmpdir, has_minimal_dependencies):
```

Comment:
Tests are great! Nit-pick, but do we need to run this and the previous test separately? Can't we combine them with `pytest.mark.parametrize`, and include another column with the expected name? In my opinion, it wouldn't seem more messy to do it that way, unless I'm missing something.

Reply:
@bchen1116 That's true, it definitely wouldn't be more messy, I was just trying to separate the tests by function/theme for ease of retest later if necessary